vmdb/app/helpers/api_helper/normalizer.rb

```diff
@@ -91,6 +95,13 @@ def normalize_href(type, value)
       normalize_url(type, "#{type}/#{value}")
     end
 
+    #
+    # Let's filter encrypted attributes, i.e. passwords
+    #
+    def normalize_encrypted(_type, _value)
+      "[FILTERED]"
+    end
```

Comment:
Should we filter the value or just completely remove the key from the output.  I was originally thinking the latter.  Why even expose that we *have* a password column.